compiler/luci/import/src/Importer.test.cpp

```diff
@@ -16,13 +16,289 @@
+    model->operator_codes[id]->version = 1;
+    return id;
+  }
+
+  uint32_t add_buffer()
+  {
+    model->buffers.emplace_back(new circle::BufferT);
+    return model->buffers.size() - 1;
+  }
+
+  uint32_t add_float_tensor(uint32_t graph_id, const std::vector<int32_t> &shape,
+                            uint32_t buffer_id)
+  {
+    auto &graph = model->subgraphs[graph_id];
+    uint32_t idx = graph->tensors.size();
+    graph->tensors.emplace_back(new circle::TensorT);
+    graph->tensors[idx]->shape = shape;
+    graph->tensors[idx]->type = circle::TensorType_FLOAT32;
+    graph->tensors[idx]->buffer = buffer_id;
+    graph->tensors[idx]->name = std::to_string(idx);
+    graph->tensors[idx]->quantization.reset(nullptr);
+    graph->tensors[idx]->is_variable = false;
+    graph->tensors[idx]->sparsity.reset(nullptr);
+    (void)graph->tensors[idx]->shape_signature;
+    return idx;
+  }
+
+  uint32_t add_builtin_operator(uint32_t graph_id, uint32_t opcode_id,
+                                const std::vector<uint32_t> &inputs,
+                                const std::vector<uint32_t> &outputs)
+  {
+    auto &graph = model->subgraphs[graph_id];
+    auto idx = graph->operators.size();
+    graph->operators.emplace_back(new circle::OperatorT);
+    graph->operators[idx]->opcode_index = opcode_id;
+    graph->operators[idx]->inputs.assign(inputs.begin(), inputs.end());
+    graph->operators[idx]->outputs.assign(outputs.begin(), outputs.end());
+    graph->operators[idx]->builtin_options.Reset();
+    (void)graph->operators[idx]->custom_options;
+    graph->operators[idx]->custom_options_format = circle::CustomOptionsFormat_FLEXBUFFERS;
+    (void)graph->operators[idx]->mutating_variable_inputs;
+    (void)graph->operators[idx]->intermediates;
+    return idx;
+  }
+
+  uint32_t add_plan_metadata(uint32_t buffer_id)
+  {
+    uint32_t idx = model->metadata.size();
+    model->metadata.emplace_back(new circle::MetadataT);
+    model->metadata[idx]->name = "ONE_execution_plan_table";
+    model->metadata[idx]->buffer = buffer_id;
+    model->buffers[buffer_id]->data.resize(4);
+    return idx;
+  }
+
+  void add_plan_entry(uint32_t plan_buffer_id, uint32_t execution_order,
+                      const std::vector<uint32_t> &offsets)
+  {
+    auto &buffer = model->buffers[plan_buffer_id]->data;
+    auto old_size = buffer.size();
+    assert(old_size % 4 == 0);
+    assert(old_size > 0);
+
+    // Allocate space for new entry
+    // +4 from entry id, +4 from entry size, +4 from execution order + offsets.size() * 4 for
+    // offsets
+    buffer.resize(old_size + 12 + offsets.size() * 4);
+    uint32_t *number_of_entries_ptr = reinterpret_cast<uint32_t *>(buffer.data());
+    *number_of_entries_ptr += 1;
+
+    uint32_t *entry_data_ptr = reinterpret_cast<uint32_t *>(buffer.data() + old_size);
+
+    entry_data_ptr[0] = *number_of_entries_ptr - 1; // entry id
+    entry_data_ptr[1] = 1 + offsets.size();         // entry size
```

Comment:
Q. Could you please explain why +1?

Reply:
This value contains size of `uint32_t` values in entry:
- 1 value is `execution_order`
- rest of the values are `offsets`

+1 is for `execution_order`